.github/workflows/build-images-ci.yaml

```diff
@@ -326,3 +334,15 @@ jobs:
         run: |
           cd image-digest/
           find -type f | sort | xargs -d '\n' cat
+
+  push-chart:
+    name: Push dev chart
+    needs: build-and-push-prs
+    permissions:
+      contents: read
+      pull-requests: read
+      statuses: write
+    uses: ./.github/workflows/push-chart-ci.yaml
```

Comment:
Which `./.github/workflows/push-chart-ci.yaml` is used here? From the default branch, the base branch or the branch from the PR? We need to think on the case once these changes go to the stable branches.

Reply:
Based on https://docs.github.com/en/actions/sharing-automations/reusing-workflows#calling-a-reusable-workflow:
> the called workflow is from the same commit as the caller workflow.